addons/binding/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/internal/eep/D2_01/D2_01.java

```diff
@@ -141,7 +142,7 @@ protected State getEnergyMeasurementData() {
 
             switch (bytes[1] >>> 5) {
                 case 0:
-                    factor /= 3600.0;
+                    factor /= (3600 * 1000);
```

Comment:
This looks like a breaking change. Can you please reason about why this is necessary?

Reply:
Sure. The result number is requested to be in kilowatt hours (kWh). When the "unit" field (bytes[1] >>> 5) has the value 0x00, the documentation says that the value is given in watt seconds (Ws). So when we divide the value by only 3600 we get watt hours (Wh). Another division by 1000 gets us the kWh.

Reply:
Sounds very reasonable. Might help if you add that knowledge as a comment to this line so that fellow developers are immediately up to speed. WDYT?

Reply:
Yup, I have added the comments :-)